lighthouserc.js

```diff
@@ -14,7 +14,9 @@ module.exports = {
 			]
 		},
 		upload: {
-			target: 'temporary-public-storage',
+			target: 'lhci',
+			serverBaseUrl: 'https://lighthouse-ci-kiva-dev.herokuapp.com/',
+			token: 'LHCI_DEV_BUILD_TOKEN goes here',
```

Comment:
If we pass a secret here, LHCI won't be able to run from PRs coming outside of our organization. Adding it in in plaintext here is probably OK but wanted to double check @mcstover @emuvente 

From https://github.com/GoogleChrome/lighthouse-ci/blob/main/docs/server.md#security - 
"The build token allows a user to upload new data to a particular project, but does not allow the destruction or editing of any historical data. If your project is open source and you want to collect Lighthouse data on pull requests from external contributors then you should consider the build token to be public knowledge."

Reply:
I have the build and admin tokens generated, just wasn't sure if we wanted to keep this build token secret or not.

Reply:
If someone uploads junk data then we will be able to delete it, correct? If so I think it's ok to include the build token. If it's abused in someway we can regenerate the build token and keep it secret after that.